scss/base/_base.scss

```diff
@@ -1,34 +1,21 @@
-/* Resetting */
-*,
-*::before,
-*::after {
-  box-sizing: border-box;
-  margin: 0;
-  padding: 0;
-}
-
-/* Setting the value of 1rem */
 html {
-  /* 1rem = 10px */
```

Comment:
- [ ] **[REQUIRED CHANGES]**

- Please, kindly remove unused code, it makes your codebase not clean.